Index: src/main/java/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>public class Main {\n    static void main(String[] args) {\n        Race race = new Race();\n        Snail s1 = new Snail(\"달팽이1\", race);\n        Snail s2 = new Snail(\"달팽이2\", race);\n        Snail s3 = new Snail(\"달팽이3\", race);\n\n        s1.start();\n        s2.start();\n        s3.start();\n\n        try {\n            s1.join();\n            s2.join();\n            s3.join();\n        } catch(InterruptedException e) {\n            e.printStackTrace();\n        }\n        race.printRanking();\n\n    }\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/Main.java b/src/main/java/Main.java
--- a/src/main/java/Main.java	(revision 2e52149daa3971ba4adc3aa3b5b91610981cf5d1)
+++ b/src/main/java/Main.java	(date 1780969711657)
@@ -8,6 +8,7 @@
         s1.start();
         s2.start();
         s3.start();
+        
 
         try {
             s1.join();
